api/test/run_test.sh

```diff
@@ -18,8 +18,6 @@ succeeded=$?
 if [[ $succeeded -eq 0 ]]; then
 	printf "${green}:-) All tests passed${endcol}\n"
 else
-  printf "Full system logs:\n"
-  docker-compose logs
```

Comment:
I know we have another ticket for this - should we hold off on this change for now?

Reply:
Oh sure. I think I did this in rage at not being able to debug the tests, didn't mean to commit it though.

Reply:
Put back in 753182c436281804b2e8508210da8c01f1ac32fc